tests/foreman/cli/test_rhcloud_inventory.py

```diff
@@ -371,3 +371,36 @@ def test_positive_generate_all_reports_job(target_sat):
         assert task_output[0].result == "success"
     finally:
         target_sat.update_setting('allow_auto_inventory_upload', True)
+
+
+@pytest.mark.rhel_ver_list([7, 8, 9])
```

Comment:
We could use @damoore044's floating selection here to pick latest 3 RHELs.
```suggestion
@pytest.mark.rhel_ver_match('N-2')
```